wa/workloads/speedometer/__init__.py

```diff
@@ -0,0 +1,62 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import re
+
+from wa import UiautoWorkload, Parameter
+from wa.framework.exception import ValidationError
+from wa.utils.types import list_of_strs
+from wa.utils.misc import unique
+
+
+class speedometer(UiautoWorkload):
```

Comment:
`speedometer` class name should be capitalised for consistency with other workloads.